builder.config.js

```diff
@@ -54,7 +54,7 @@ const config = {
     {
       provider: 's3',
       bucket: 'downloads.mapeo.app',
-      path: '/desktop'
+      path: process.env.ARCH === 'ia32' ? '/desktop-ia32' : '/desktop'
```

Comment:
Did a local build with this and confirmed that the generated `app-update.yml` file had the following:

```yml
path: /desktop-ia32
```

Reply:
Also open to changing the directory name if there are any suggestions

Reply:
```suggestion
      path: process.env.ARCH === 'ia32' ? '/desktop-win-ia32' : '/desktop'
```

Since this is specifically windows (whereas the other path has all platforms)